hacksquad_discordpy/wrappers/api_thingies.py

```diff
@@ -0,0 +1 @@
+hacksquad_endpoint = 'https://www.hacksquad.dev/api/'
```

Comment:
File should be just "constants.py". All variables must be in upper case (I.e. HACKSQUAD_ENDPOINT = ...)
Preferably, remove the `/` at the end to put in requests so that they make sense (`{BASE_URL}/teams` rather than `{BASE_URL}teams`)